tests/simulate-chime/test_client.py

```diff
@@ -223,7 +243,14 @@ def test_client():
         # check beam offset in config
         assert conf["frb"]["update_beam_offset"]["beam_offset"] == 10
 
+        # check frb gain dir in config
+        assert conf["frb_gain"]["frb_gain_dir"] == "insert/sth/useful"
+
+        # check pulsar gain dir in config
+        assert conf["pulsar_gain"]["pulsar_gain_dir"] == ["insert/sth/useful"]
+
     # TODO: check receiver config: bad inputs, gains
+    # TODO check status: timestamp of frb-gain-dir
```

Comment:
This is already tested in `test_wait.py`, no?

Reply:
This is to test if the config actually works with kotekan.